clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ubam/UbamQueryInput.scala

```diff
@@ -22,6 +22,7 @@ case class UbamQueryInput(
   baitIntervals: Option[Symbol] = None,
   targetIntervals: Option[Symbol] = None,
   aggregatedBy: Option[AggregatedBy] = None,
```

Comment:
I'd remove `aggregatedBy`

Reply:
I think I'm going to do that in another PR.

Reply:
~~Will that create issues when we try to upsert existing documents that have an `aggregatedBy` since they'll have the key but it won't map anything?~~ Nevermind, this is the query input.